cli/cmd/cli_state.go

```diff
@@ -430,6 +430,8 @@ func (c *cliState) envs() []string {
 		fmt.Sprintf("LW_NOCOLOR=%s", os.Getenv("NO_COLOR")),
 		fmt.Sprintf("LW_LOG=%s", c.LogLevel),
 		fmt.Sprintf("LW_JSON=%v", c.jsonOutput),
+		fmt.Sprintf("LW_PROFILE=%s", c.Profile),
+		fmt.Sprintf("LW_DEBUG=%v", c.LogLevel == "DEBUG"),
```

Comment:
Instead of `LW_DEBUG` I proposed to pass `LW_LOG`.

`LW_LOG` is better since we could pass different log levels such as `INFO`, `DEBUG`, etc.

The `--debug` flag was introduced before we created the [lwlogger](https://github.com/lacework/go-sdk/tree/main/lwlogger) Go package. 

Do you still think we should pass both `LW_DEBUG` and `LW_LOG`?

Reply:
It's better to pass both I think on the idea that we forward all the global flags as environment variables.

I do have some concern with what happens when the CLI adds TRACE.  One answer is that it never does, which is fine.  But LW_DEBUG=true is very clear, and the CLI can still set LW_DEBUG=true even if it adds LW_LOG=TRACE.

Reply:
FWIW the `remediate` component just ended up using `LW_LOG`:
```
Log:        lwlogger.New("").Sugar(), // get from "LW_LOG"
```

And when I got my api client I used:
```
	c.Log.Debugw("initializing lacework api client")
	writer := &zapio.Writer{Log: c.Log.Desugar(), Level: zapLogLevel(c.LogLevel)}
	defer writer.Close()

	// Initialize Lacework API client
	apiopts := []api.Option{
		api.WithSubaccount(c.Subaccount),
		api.WithApiV2(),
		api.WithLogLevelAndWriter(c.LogLevel, writer),
	}
```